src/main/java/Duke.java

```diff
@@ -1,10 +1,62 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String greeting = "    ____________________________________________________________\n" +
+                "     Hello! I'm Duke\n" +
+                "     What can I do for you?\n" +
+                "    ____________________________________________________________\n";
+        String bye = "     Bye. Hope to see you again soon!\n" +
+                "    ____________________________________________________________";
+        String line = "    ____________________________________________________________\n";
```

Comment:
Consider declaring this variable before you declare greeting, and make use of it.